### PR TITLE
Hyundai: add 2020 Palisade FW versions

### PR DESCRIPTION
**Pull request by d3v1n11**
Add missing FW versions for 2020 Hyundai Palisade

### CODE REVIEW

**Comment by vanillagorillaa** [comment]
Be sure to include a route that these firmware versions were taken from.

**Comment by d3v1n11** [comment]
@vanillagorillaa how do I include the route in the pull request? Sorry, this is the first time I've done one.

**Comment by d3v1n11** [comment]
Is this all you need?

e84e3c263d62889f|2022-10-19--11-59-51

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -809,7 +812,9 @@ class Buttons:
       b'\xf1\x00LX2 MFC  AT USA LHD 1.00 1.00 99211-S8110 210226',
     ],
     (Ecu.transmission, 0x7e1, None): [
+      b'\x00\x00SLX4G38NB3X\xa8\xc08',
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:815]:
```suggestion
```

File: selfdrive/car/hyundai/values.py [hunk 2]
```diff
@@ -825,6 +830,7 @@ class Buttons:
       b'\xf1\x87LDLVAA5194534HH1ffvguUUUx\x88\xa7\x88h\x99\x96\x89x\x88\x97\x88ro\xf9\xff\x98wo\xff\xaaM\xf1\x81U903\x00\x00\x00\x00\x00\x00\xf1\x00bcsh8p54  U903\x00\x00\x00\x00\x00\x00TON4G38NB2[v\\\xb6',
       b'\xf1\x87LDLVAA5949924HH1\xa9\x99y\x97\x87wwwx\x99\x97\x89x\x99\xa7\x89x\x99\xa7\x89\x87_\xfa\xffeD?\xff\xf1\xfd\xf1\x81U903\x00\x00\x00\x00\x00\x00\xf1\x00bcsh8p54  U903\x00\x00\x00\x00\x00\x00TON4G38NB2[v\\\xb6',
       b'\xf1\x87LDLVBN560098KF26\x86fff\x87vgfg\x88\x96xfw\x86gfw\x86g\x95\xf6\xffeU_\xff\x92c\xf1\x81U891\x00\x00\x00\x00\x00\x00\xf1\x00bcsh8p54  U891\x00\x00\x00\x00\x00\x00SLX4G38NB2\xafL]\xe7',
+      b'\xf1\x87LDLVBN597104KF26\xb9\x99\x89\x98\x97x\x97\x89y\x99\xa7\x99\x88\x99\xa8\x89\x88\x88\x98\x88\xaao',
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:833]:
```suggestion
```

File: selfdrive/car/hyundai/values.py [hunk 3]
```diff
@@ -858,6 +864,7 @@ class Buttons:
       b'\xf1\x87LDMVBN950669KF37\x97www\x96fffy\x99\xa7\x99\xa9\x99\xaa\x99g\x88\x96x\xb8\x8f\xf9\xffTD/\xff\xa7\xcb\xf1\x81U922\x00\x00\x00\x00\x00\x00\xf1\x00bcsh8p54  U922\x00\x00\x00\x00\x00\x00SLX4G38NB5\xb9\x94\xe8\x89',
       b'\xf1\x87LDLVAA4478824HH1\x87wwwvfvg\x89\x99\xa8\x99w\x88\x87x\x89\x99\xa8\x99\xa6o\xfa\xfffU/\xffu\x92\xf1\x81U903\x00\x00\x00\x00\x00\x00\xf1\x00bcsh8p54  U903\x00\x00\x00\x00\x00\x00TON4G38NB2[v\\\xb6',
       b'\xf1\x87LDMVBN871852KF37\xb9\x99\x99\x99\xa8\x88\x88\x88y\x99\xa7\x99x\x99\xa7\x89\x88\x88\x98\x88\x89o\xf7\xff\xaa\x88o\xff\x0e\xed\xf1\x81U922\x00\x00\x00\x00\x00\x00\xf1\x00bcsh8p54  U922\x00\x00\x00\x00\x00\x00SLX4G38NB5\xb9\x94\xe8\x89',
+      b'\xf6\xffEU\x7f\xff\xd6\x7f\xf1\x81U891\x00\x00\x00\x00\x00\x00\xf1\x00bcsh8p54  U891\x00\x00\x00\x00',
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:867]:
```suggestion
```

**Comment by sshane** [comment]
Make sure you only add versions for the brand of your car (Hyundai)

**Pull request merged**